src/connector/src/sink/catalog/desc.rs

```diff
@@ -104,7 +103,7 @@ impl SinkDesc {
             downstream_pk: self.downstream_pk,
             distribution_key: self.distribution_key,
             owner,
-            dependent_relations,
+            dependent_relations: vec![], // TODO(rc): to be deprecated
```

Comment:
Are we going to deprecate all `dependent_relations` fields in different catalogs? Are they referenced anywhere?

Reply:
I don't think these `dependent_relations` fields are used elsewhere except when creating meta objects. cc @yezizp2012 can you confirm?

Reply:
IIUC that's the only usage when creating meta objects. Previouly the process of detecting dependency cycles for `sink into table` relies on it in v1, but that functionality has been transferred to the meta side. However, the code related to this in the FE remains and can be removed for clarity.

Reply:
btw, @shanicky will rise a PR to clean that part.